doc/contribute/committer_expectations.rst

```diff
@@ -0,0 +1,165 @@
+  :ref:`commit_message_guidelines`.
+
+- All files in the PR must comply with :ref:`Licensing
+  Requirements<licensing_requirements>`.
+
+- Follow the Zephyr :ref:`coding_style`.
+
+- PRs must pass all CI checks. This is a requirement to merge the PR, but you
+  may mark a PR as draft and explicitly request reviewers to provide early
+  feedback.
+
+- When breaking a PR into multiple commits, each commit must build cleanly. This
+  is not enforced by the CI system, so it is the PR author’s responsibility to
+  verify.
+
+- When major new functionality is added, tests for the new functionality MUST be
+  added to the automated test suite. All new APIs MUST be documented and tested
+  and tests MUST cover at least 80% of the added functionality using the code
+  coverage tool and reporting provided by the project..
+
+- Incompatible changes to APIs must also update the release notes for the
+  next release detailing the change.  APIs marked as experimented are excluded
+  from this requirement. [#api-updates]_
+
+- PRs must also satisfy all :ref:`merge_criteria` before a member of the release
+  engineering team will merge the PR into the zephyr tree.
+
+Maintainers may request you break up a PR into smaller PRs and may request you
+create an :ref:`RFC proposal <rfcs>`.
+
+Workflow Suggestions That Help Reviewers
+========================================
+
+- Don’t resolve comments until the question has been answered or the request
+  completed in an update to the PR.
+
+- Try to minimize rebases in the middle of a review. If a rebase is required,
+  push this as a separate update with no functional changes to the PR. See the
+  note below.
+
+- Instead of amending commits, append fixup commits to the PR to address
+  reviewer comments. This provides reviewers a clear history of the changes made
+  to the PR, along with the rationale for changes. Prior to the final approval
+  and merge of the PR, you must perform a squash merge to cleanup the commit
+  history. (TBD - This needs a PoC to demonstrate this workflow).
+
+Reviewer Expectations
+#####################
+
+- Be respectful when commenting on PRs. Refer to the Zephyr `Code of Conduct`_
+  for more details.
+
+- Review PRs within a reasonable time. (TBD - define soft guidelines for
+  “reasonable”)
+
+- Try to provide feedback on the entire PR in one shot. This provides the author
+  an opportunity to address all comments in the next PR update.
+
+- Partial reviews which focus on a domain (e.g. Devicetree) are permitted, but
+  the reviewer must add a comment indicating what portion of the PR they
+  reviewed. (TBD - ideally we could designate multiple assignees on a PR to
+  cover cross-domain PRs).
+
+- Avoid increasing scope of the PR by requesting new features, especially when
+  there is a corresponding :ref:`RFC <rfcs>` associated with the PR. Instead add
+  your suggestions as a comment to the :ref:`RFC <rfcs>`. This also encourages
```

Comment:
I think this is an excellent mechanism for avoiding drive-by feature creep requests

Reply:
The opposite may be good: ask to decrease the scope of the PR if it makes sense, maybe split some commit on a separate PR.

Reply:
+1

Reply:
Regarding decreasing scope, that is mentioned above just below the PR requirements.  Should I move that into the reviewer expectations.